sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3778,3 +3778,37 @@ test_suite(
     tags = ["manual"],
     tests = ["rom_ext_upgrade_interrupt_{}".format(lc_state) for lc_state, _ in get_lc_items()],
 )
+
+opentitan_functest(
+    name = "test_openocd",
+    srcs = ["empty_test.c"],
+    cw310 = cw310_params(
+        bitstream = ":bitstream_default_otp",
```

Comment:
Just an FYI: this bitstream loads the OTP [image](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/e2e/BUILD;drc=31a5c9c9ad75ce89c6fa67cb5b6237722472b2bb;l=112) that puts the HW in the RMA LC state, but the line below uses the `PROD` signing key? Since it looks like you never want to boot the flash image I'm guessing this doesn't matter? Just wanted to point this out.

Reply:
This was just a super ugly hack to get bazel to run the test. As far I know, there is no test function that just loads a bitstream, the functest always wants some flash image even though I don't need any code running for the LC TAP. But yeah I should probably change that :)